core/lib/cc/tmux.go

```diff
@@ -100,13 +92,7 @@ func (pane *Emp3r0rPane) TmuxRespawn() (err error) {
 // TmuxPrintf like printf, but prints to a tmux pane/window
 // id: pane unique id
 func (pane *Emp3r0rPane) TmuxPrintf(clear bool, format string, a ...interface{}) {
-	if pane == nil {
```

Comment:
if pane is nil , function call will panic. so , pane never be nil if function called.